### DISCUSSION TITLE
Linux- Debian, Raspbian Bookworm, Ubuntu 24.04 RustDesk- service headless

### DISCUSSION THREAD

**Discussion by IT-Betreuung** [rustdesk/rustdesk#7895] [General]
Unfortunately, the RustDesk- service no longer starts in headless mode,
I first have to register a user via Remote Desktop,
then RustDesk is "active" again
((sometimes several times(I mean the RustDesk- service and a instance)),
previously I have had to register a user via Remote Desktop,
because of missing X11 or Wayland/ Display.

When installing on a "fresh" machine (Raspbian and Debian),
but also when updating I get the following warning:
„Warning: The unit file, source configuration file or drop-ins of rustdesk.service changed on disk. Run 'systemctl daemon-reload' to reload units.“

For the following assets:
rustdesk-1.2.4-x86_64.deb
rustdesk-1.2.4-armv7-sciter.deb
rustdesk-1.2.4-aarch64.deb

**Comment by rustdesk** [comment]
Today, we did change this unit file 2-3 hours ago. https://github.com/rustdesk/rustdesk/commit/2ff3b24b33e124593654f0c33861ceb41ef145c9

**Reply by IT-Betreuung** [reply]
Ubuntu, Raspbian & Debian,
no Service,
a user has to be logged in.

sudo systemctl enable rustdesk.service
and
sudo systemctl daemon-reload
and
sudo systemctl restart rustdesk
or
sudo systemctl start rustdesk.service
does not help.
